recipe/patches/pre-7.3.7-004-sysconfigdata-env.patch

```diff
@@ -1,5 +1,5 @@
---- lib-python/3/distutils/sysconfig_pypy.py.bak        2021-10-29 09:42:09.465488143 -0500
-+++ lib-python/3/distutils/sysconfig_pypy.py    2021-10-29 09:43:07.029386160 -0500
+--- a/lib-python/3/distutils/sysconfig_pypy.py	2021-10-29 09:42:09.465488143 -0500
++++ b/lib-python/3/distutils/sysconfig_pypy.py	2021-10-29 09:43:07.029386160 -0500
```

Comment:
@isuruf: I think patches should have a tab after the file name to make the regex in conda-build/source.py::_get_patch_file_details happy. Also it seems to like starting with an `a/`, `b/` better. Maybe not needed, but it seems to avoid the `"Patch level ambiguous, selecting least deep"` conflict tests